package.json

```diff
@@ -33,12 +33,20 @@
 	],
 	"devDependencies": {
 		"ava": "*",
+		"esm": "3.1.1",
 		"semver": "^5.3.0",
 		"xo": "*"
 	},
 	"xo": {
 		"ignores": [
 			"test*.js"
 		]
+	},
+	"ava": {
+		"babel": false,
```

Comment:
By disabling Babel and falling back to ESM, we are preserving the actual input as is (i.e. trailing comma stays when `fn.toString()` is called).